src/Sulu/Bundle/ContentBundle/DependencyInjection/SuluContentExtension.php

```diff
@@ -67,6 +67,19 @@ public function prepend(ContainerBuilder $container)
                 ]
             );
         }
+
+        if ($container->hasExtension('fos_rest')) {
+            $container->prependExtensionConfig(
+                'fos_rest',
+                [
+                    'exception' => [
+                        'codes' => [
+                            'Sulu\Component\Content\Exception\ResourceLocatorAlreadyExistsException' => 409,
+                        ],
```

Comment:
you have removed more than this exception shouldn´t we add here more exceptions?